iset.mm

```diff
@@ -129911,11 +129911,16 @@ then the Limited Principle of Omniscience (LPO) implies excluded middle.
     ctiunct.a $e |- ( ph -> F : _om -onto-> ( A |_| 1o ) ) $.
     ctiunct.b $e |- ( ( ph /\ x e. A )
       -> G : _om -onto-> ( B |_| 1o ) ) $.
-    $( The union of countably many countable sets is countable.  The
-       construction here is constructive in the sense that the hypothesis
-       provides a specific class for ` G ` in terms of ` x ` , as opposed to a
-       construction of the form ` A. x e. A E. g ` .  Countable choice would be
-       needed to turn the latter into the former.
+    $( A sequence of enumerations gives an enumeration of the union.  We refer
+       to "sequence of enumerations" rather than "countably many countable
+       sets" because the hypothesis does not provide countability for each
+       ` B ( x ) ` in isolation (independent of how it is enumerated), but in
+       terms of the ` B ( x ) ` together with the ` G ( x ) ` which enumerates
+       it.  Countable choice would be needed to turn the "countably many
+       countable sets" version into the "sequence of enumerations" version.
```

Comment:
Maybe "To obtain a sequence of enumerations from countably many enumerations requires the axiom of countable choice." ?

Reply:
At least to my eyes that wording would draw attention away from the existential quantifier in the unct-style (as opposed to ctiunct-style) version.

The existential quantifier is part of what we mean by countability in places like https://us.metamath.org/ileuni/ctm.html and https://us.metamath.org/ileuni/ctssdc.html

Reply:
I agree. I actually wanted to be more precise. In the expression "countably many countables", you have a non-constructive (or "non-explicit") notion at two places, and in "enumeration of enumerations", you have a constructive notion at two places. Therefore, to go from one to the other, you have two steps. The expression I wrote above, "contably many enumerations", is midway. And I think the critical step, the one which requires the axiom of coutable choice, is the step from that midway to the "enumeration of enumerations" version. I hope this makes sense ? (and that it is correct !)

Reply:
Take a look with a fresh pair of eyes (that is, not tonight) - for one thing I did make a revision here after you wrote that.

Reply:
The various comments since last time clarified things, and therefore the replacement I proposed at the top of this thread is indeed more precise (namely, "To obtain a sequence of enumerations from countably many enumerations requires the axiom of countable choice.")